tensorflow/lite/experimental/swift/BUILD.apple

```diff
@@ -102,7 +153,7 @@ swift_library(
     tags = TFL_DEFAULT_TAGS + ["manual"],
     deps = [
         ":Resources",
-        ":TensorFlowLite",
+        ":TensorFlowLiteAllDelegates",
```

Comment:
Would be fine with reverting this change as the test app doesn't currently reference any delegates and I don't think there are plans to due to.